include/faker-cxx/Helper.h

```diff
@@ -191,7 +191,7 @@ class Helper
             throw std::invalid_argument{"Sum of weights is zero."};
         }
 
-        const std::integral auto targetWeightValue = Number::integer<unsigned>(1, sumOfWeights);
+        const std::integral auto targetWeightValue = faker::number::integer<unsigned>(1, sumOfWeights);
```

Comment:
it could be without faker prefix as it is in faker namespace, right?

Reply:
yes, sorry, i changed it now